dove/src/cli.rs

```diff
@@ -90,6 +91,12 @@ pub enum DoveCommands {
         #[structopt(flatten)]
         cmd: Deploy,
     },
+    #[structopt(about = "Managing secret phrases")]
```

Comment:
Is it really about "managing secret phrases" now? Is there maybe a better way to describe what command does?